tests/js/reactions.js

```diff
@@ -181,6 +246,72 @@ suite('Custom Element Reactions', function() {
 
   });
 
+  suite('customized built-in elements', function() {
+    test('inherits native behavior such as a button\'s disabled property', function(done) {
```

Comment:
Tests that you can actually get the native behavior for specific elements (like `disabled` for buttons) to work just like it's native.